packages/frontend-api/src/Model/Store/StoreRepository.php

```diff
@@ -88,6 +97,28 @@ public function getFilteredQueryBuilder(
             $queryBuilder->setFirstResult($offset);
         }
 
-        return $queryBuilder->getQuery()->getResult();
+        $results = $queryBuilder->getQuery()->getResult();
+
+        if ($storesFilterOptions->getCoordinates() === null) {
+            return $results;
+        }
+
+        $stores = [];
+
+        foreach ($results as $storeArray) {
+            /** @var \Shopsys\FrameworkBundle\Model\Store\Store $store */
+            $store = $storeArray[0];
+
+            /** @var string|null $distance */
+            $distance = $storeArray['distance'] ?? null;
+
+            if ($distance !== null) {
+                $store->setDistance((int)$distance);
```

Comment:
when I get a store elsewhere (eg. by `getByUuid` method), will the distance be empty?

Reply:
Yes precisely. But it's not hard to implement it on your own when you need it. Or do you have some specific idea of how and where this should work?

Reply:
to avoid confusion, I would probably add it to resolver map, at least in API it will be always present